src/lib/nats/nats.ts

```diff
@@ -0,0 +1,7 @@
+import { NATS_URL } from "../../config";
+import { NatsConnection } from "nats";
+
+export function drain(natsConnection: NatsConnection): Promise<void> {
+    console.log(`[AIRLOCK] Draining connection to NATS server ${NATS_URL}`);
```

Comment:
can we get the url from the connection itself and avoid using the config ?

Reply:
Yes, absolutely.